### DISCUSSION TITLE
the aircraft start in the runway each time

### DISCUSSION THREAD

**Discussion by DemonRem** [flybywiresim/aircraft#5777] [Help]
hey guys i using the a32nx and from some reason the aircraft spawn on the runway even i choose on the gate 
also please check this flight plan if its correct
[LLBGRJTT.zip](https://github.com/flybywiresim/a32nx/files/7107556/LLBGRJTT.zip)

**Comment by robinottawa** [comment]
Still happening? It did the same for me but it seemed to have something to do with update 5 and loss of capabilities in the world flight planning tool in msfs. Mine works again. I think.
